index.html

```diff
@@ -8,9 +8,46 @@
     <link rel="stylesheet"  href="style.css">
 </head>
 <body>
-    <header>
-        
+    <div class ="porfolio_container">
+    
+    <header >
+       
+        <nav class = "header_navigation">
+            <ul class ="item_navigation">
+                <li class ="item"><a class= "text" href="">Omotayo Akande</a></li>
```

Comment:
- [ ] As requested by the previous reviewer your logo should wrap with `a` tag for better semantic HMTL.